e2e-tests/pages/insurance/eligibility/memberOfAGroup.js

```diff
@@ -0,0 +1,11 @@
+import { INSURANCE_FIELD_IDS } from '../../../constants/field-ids/insurance';
+
+const { MEMBER_OF_A_GROUP } = INSURANCE_FIELD_IDS.ELIGIBILITY;
+
+const memberOfAGroupPage = {
+  summary: () => cy.get(`[data-cy="${MEMBER_OF_A_GROUP}-description"] summary`),
+  details: () => cy.get(`[data-cy="${MEMBER_OF_A_GROUP}-description"]`),
+  description: () => cy.get(`[data-cy="${MEMBER_OF_A_GROUP}-description-details"]`),
+};
+
+export default memberOfAGroupPage;
```

Comment:
Should this be a partial, since the content strings are not eligibility specific 

Similar to `creditPeriodWithBuyer`, incase we need to use in another area

Reply:
done